src/components/swapv2/LimitOrder/LimitOrderForm.tsx

```diff
@@ -630,45 +620,44 @@ const LimitOrderForm = function LimitOrderForm({
 
   const warningMessage = useMemo(() => {
     const messages = []
+
     if (currencyIn && displayRate && !deltaRate.profit && deltaRate.percent) {
       messages.push(
         <Text>
           <Trans>
-            Your limit order price is{' '}
-            <Text as="span" fontWeight={'500'} color={theme.warning}>
-              {deltaRate.percent}
-            </Text>{' '}
-            worse than the current market price
+            Your limit order price is <HightLight>{deltaRate.percent}</HightLight> worse than the current market price
           </Trans>
         </Text>,
       )
     }
-    const thresHold = chainId === ChainId.MAINNET ? 300 : 10
-    if (outputAmount && estimateUSD.rawInput && estimateUSD.rawInput < thresHold) {
+
+    const isMainNet = chainId === ChainId.MAINNET
+    const thresHold = isMainNet ? NUMBERS.ETH_USD_THRESHOLD : NUMBERS.REST_USD_THRESHOLD
```

Comment:
after change the constant is object config by chainId, update this to:
`const threshold = USD_THRESHOLD[chainId]`
btw `threshold`, not `thresHold` :D

Reply:
updated sir